### DISCUSSION TITLE
Web ui

### DISCUSSION THREAD

**Discussion by rj10rd** [arendst/Tasmota#13727] [Q&A] [answered]
Can i access web ui using hostname if yes please explain how?

**Comment by barbudor** [comment, marked as answer]
You need your router to register the hostname announced by Tasmota and serve the IP address when a DNS request is made
Not Tasmota dependant.

**Reply by barbudor** [reply]
Alternatively, if you are using fixed IP address, you can also edit the `hosts` file of your computer to "teach" it the translattion between the name and the IP address. You need to do this on every computer that needs to access tasmota from its name

**Reply by rj10rd** [reply]
Mdns is related with my question?

**Reply by sfromis** [reply]
mDNS is by default not included in Tasmota. You can optionally configure this bad option at build time. My router runs dnsmasq to automatically provide DNS lookup based on host names.

**Reply by rj10rd** [reply]
Why u say bad option and what custom # define use for mdns

**Reply by sfromis** [reply]
Bad due to the Arduino Core implementation having caused lots of trouble, leading to the Tasmota project cutting the losses by giving up on including it as a standard feature. Better to use the flash space for other features. `USE_DISCOVERY` is the #define.
